website/static/js/addonNodeConfig.js

```diff
@@ -0,0 +1,472 @@
+/**
```

Comment:
Same as above, this file should come from https://github.com/CenterForOpenScience/osf.io/pull/2014